### BIG differences, this is diff of wrong files.
### CLUS "System Class NULL"
### and CLHS topic selection http://clhs.lisp.se/Body/a_null.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:null.txt	2017-04-27 22:01:04.000000000 +0100
+++ saved-pages-clus/cl:types:null.txt	2017-04-27 22:01:04.000000000 +0100
@@ -1,4 +1,19 @@
-NULL
-Please select which reference to NULL you intended
+System Class NULL

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+null, symbol, list, sequence, t
+Description
+The only object of type null is nil, which represents the empty list and can also be notated '().
+See Also
+{\secref\SymbolTokens}, {\secref\LeftParen}, {\secref\PrintingSymbols}
\ No newline at end of file
